---a/net/minecraft/world/biome/BiomeGenEnd.java
+++b/net/minecraft/world/biome/BiomeGenEnd.java
@@ -2,15 +2,12 @@
 
 import net.minecraft.entity.monster.EntityEnderman;
 import net.minecraft.init.Blocks;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class BiomeGenEnd extends BiomeGenBase
-{
+
+public class BiomeGenEnd extends BiomeGenBase {
+
     private static final String __OBFID = "CL_00000187";
 
-    public BiomeGenEnd(int p_i1990_1_)
-    {
+    public BiomeGenEnd(int p_i1990_1_) {
         super(p_i1990_1_);
         this.spawnableMonsterList.clear();
         this.spawnableCreatureList.clear();
@@ -21,10 +21,4 @@
         this.fillerBlock = Blocks.dirt.getDefaultState();
         this.theBiomeDecorator = new BiomeEndDecorator();
     }
-
-    @SideOnly(Side.CLIENT)
-    public int getSkyColorByTemp(float p_76731_1_)
-    {
-        return 0;
-    }
-}
+}
